### PR TITLE
This are not yet committed to docs.microsoft site

### PR DESCRIPTION
**Pull request by Malind19**
balacne rename to balance. Checked here and its all good, need to publish to Docs stie.

### CODE REVIEW

File: docset/windows/dhcpserver/add-dhcpserverv4failover.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ This example creates a hot standby, or active-passive, failover relationship bet
 These scopes will be created on the partner DHCP server service that runs on the computer named dhcpserver2.contoso.com as part of the failover relationship creation.
 The DHCP server service that runs on the computer named dhcpserver.contoso.com will be the standby DHCP server service and the DHCP server service that runs on the computer named dhcpserver2.contoso.com will be the active DHCP server service in the failover relationship.
 
-### Example 3: Create an active-active failover relationship with specified load balance amount
+### Example 3: Create an active-active failover relationship with specified load balance amount
```
Comment (nschonni) [inline docset/windows/dhcpserver/add-dhcpserverv4failover.md]: Looks like this is just adding trailing pace

Comment (kenwith) [reply docset/windows/dhcpserver/add-dhcpserverv4failover.md]: I think this must have already been fixed in your PR @nschonni .. I don't think we need a trailing space so will close this one. Please re-open if I am incorrect.

Comment (nschonni) [reply docset/windows/dhcpserver/add-dhcpserverv4failover.md]: Actually it looks like it was fixed in https://github.com/MicrosoftDocs/windows-powershell-docs/commit/e440a817f1798f8946ee1e6ca431b56053a9d38f but I thing this PR was just making a change to try and trigger the page to be republished

Comment (Malind19) [reply docset/windows/dhcpserver/add-dhcpserverv4failover.md]: Yes, nothing to be done on my end. added the space as I had to make a pull request. Doc site publishing was required and mentioned in the comment above.

**Pull request closed without merging**